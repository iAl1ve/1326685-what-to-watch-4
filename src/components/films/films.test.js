import React from "react";
import renderer from "react-test-renderer";
import Films from "./films.jsx";

const onTitleButtonClick = () => {};

describe(`Test Render Films`, () => {
  it(`Render Films`, () => {
    const tree = renderer
      .create(<Films
        title = {`Moonrise Kingdom`}
        onTitleButtonClick = {onTitleButtonClick}
      />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
