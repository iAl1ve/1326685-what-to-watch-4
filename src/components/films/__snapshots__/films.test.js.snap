// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Render Films Render Films 1`] = `
<article
  className="small-movie-card catalog__movies-card"
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Moonrise Kingdom"
      height="175"
      src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
    onClick={[Function]}
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Moonrise Kingdom
    </a>
  </h3>
</article>
`;
